charts/loki/templates/servicemonitor.yaml

```
@@ -32,4 +32,11 @@ spec:
     {{- if .Values.serviceMonitor.path }}
     path: {{ .Values.serviceMonitor.path }}
     {{- end }}
+    {{- if .Values.serviceMonitor.scheme }}
+    scheme: {{ .Values.serviceMonitor.scheme }}
+    {{- end }}
+    {{- if .Values.serviceMonitor.tlsConfig }}
+    tlsConfig: 
+{{ toYaml .Values.serviceMonitor.tlsConfig | indent 6 }}
+    {{- end }}
```

Better with `with` as demo default value `{}` in `values.yaml`.

```suggestion
    {{- with .Values.serviceMonitor.tlsConfig }}
    tlsConfig: 
        {{- toYaml . | nindent 6 }}
    {{- end }}
```